projects/_posts/2018-09-02-PS4Explor3r.md

```diff
@@ -6,3 +6,16 @@ source_code_url: "https://gist.github.com/artpoz/d66f72a6092b580b9903d088dd6d126
 building_instructions_url: "http://robotsquare.com/2015/10/06/explor3r-building-instructions/"
 excerpt: "Lego vehicle Explor3r controlled by PS4 gamepad (dualshock 4)"
 ---
+
+Lego vehicle Explor3r controlled by PS4 gamepad (Dualshock 4)
+
+# Build
+
+Build Explor3r from these [instructions](http://robotsquare.com/2015/10/06/explor3r-building-instructions/).
+
+# Install
+
+- Pair your gamepad with EV3 by press and hold the PS button and the sharing button simultaneously.
+- Copy [ps4explor3r.py](https://gist.github.com/artpoz/d66f72a6092b580b9903d088dd6d1262) into EV3 Brick. 
+- Choose file from brickman menu or run from command line 'python3 ps4explor3r.py'.
```

Comment:
"Choose this file"

Also can use backtick ``` ` ``` instead of single quote for code formatting.

    `python3 ps4explor3r.py`